src/ceph-disk/ceph_disk/main.py

```diff
@@ -1536,9 +1546,7 @@ def zap(dev):
     """
     Destroy the partition table and content of a given disk.
     """
-    dev = os.path.realpath(dev)
```

Comment:
You must keep dev here. It is used later in the function for display  and processing. https://docs.python.org/2/library/os.path.html#os.path.realpath removes symbolic links. This is the part I'm worried about because sometime we want the symbolic link, not the device it points to.

os.path.realpath is used carefully and it is complicated to figure out exactly why and where. This is why it needs to stay exactly where it is and not be used more than it should.